doc/classes/Node.xml

```diff
@@ -719,6 +731,13 @@
 				Sets whether this is an instance load placeholder. See [InstancePlaceholder].
 			</description>
 		</method>
+		<method name="set_unique_name_in_owner">
+			<return type="void" />
+			<argument index="0" name="enable" type="bool" />
+			<description>
+				Set this name as the unique name in the owner (most of the time, the scene it belongs to). This allows the node being accessed as "%Name" instead of the full path from any node within that scene.
```

Comment:
```suggestion
				Sets this node's name as the unique name in its [member owner]. This allows the node to be accessed as [code]%Name[/code] instead of the full path, from any node within that scene.
```